packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/checkIfClassExists..st

```diff
@@ -0,0 +1,8 @@
+code browser
+checkIfClassExists: aMessageTally
```

Comment:
Functions that return boolean values usually follow a different naming scheme than other functions. Please consider renaming this function. How about `classExists: ...`?